### DISCUSSION TITLE
Is it possible to use useHover/useClick/useDismiss without React?

### DISCUSSION THREAD

**Discussion by krutoo** [floating-ui/floating-ui#2143] [Q&A] [answered]
Hi, i want to use interactions hooks without React (more precisely, directly with the DOM node) 

Does the library provide framework agnostic solutions?

**Comment by atomiks** [comment, marked as answer]
Only the positioning part of the library is framework- (well, everything-) agnostic. The interactions are only available for React DOM. Initially, when I was first prototyping it, I wanted to make it framework agnostic but it was extremely buggy and I gave up and went React-only. It's possible that in the future it will be, but not right now.

**Reply by krutoo** [reply]
Thank you for the answer
